src/OpenLoco/src/Windows/KeyboardShortcuts.cpp

```diff
@@ -178,9 +178,9 @@ namespace OpenLoco::Ui::Windows::KeyboardShortcuts
 
             if (shortcuts[i].keyCode != 0xFFFFFFFF)
             {
-                if (shortcuts[i].modifiers == 1)
+                if (shortcuts[i].modifiers == KeyModifier::shift)
```

Comment:
this code looks a little wrong but its outside of the scope of this PR. Looks like you can have a shortcut with moultiple modifiers but it wont display correctly if you do. Would need to investigate if this is actually true.

Reply:
Agreed. That code looked strange to me as well.